meMonitor

```diff
@@ -0,0 +1,35 @@
+os.loadAPI("M3C/utils")
+os.loadAPI("M3C/kvApi")
+
+local meDrive = peripheral.wrap("left")
+local modem = peripheral.wrap("back")

```

Comment:
lets use the utils here to wrap these

Reply:
Updated to use utils.findAndWrap() for both.